src/agent/src/device.rs

```diff
@@ -408,6 +416,39 @@ async fn get_vfio_pci_device_name(
     Ok(format!("{}/{}", SYSTEM_DEV_PATH, &uev.devname))
 }
 
+#[cfg(target_arch = "s390x")]
+#[derive(Debug)]
+struct VfioApMatcher {
```

Comment:
This is analogous to `PciMatcher` *not* to `VfioMatcher` which you renamed to `VfioPciMatcher`.